lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -621,15 +627,21 @@ def write(chunk)
           severity = compute_severity(
             entry_resource.type, record, entry_common_labels)
 
+          # Extract "trace" as LogEntry field according to specified config
```

Comment:
Done. Also rewrote this comment a bit after the rename of the variable to `trace_resource`.